packages/embeds/embed-core/src/embed.ts

```diff
@@ -458,6 +480,12 @@ class CalApi {
       },
     });
 
+    // If someone re-executes inline embed instruction, we want to ensure that duplicate inlineEl isn't added to the page per namespace
+    if (this.cal.inlineEl && document.body.contains(this.cal.inlineEl)) {
+      console.warn("Inline embed already exists. Ignoring this call");
```

Comment:
Adding it back what I reverted here https://github.com/calcom/cal.com/pull/13637/files#diff-e895c26df52266ecfd6334060b1dc6fa05cac4106eee7f713d0a07472cc08d07

Now I am checking for the presence of inlineEl in DOM instead of just checking the property which could be out-of-sync with DOM. So, it seems safe to add it back.